srml/consensus/src/lib.rs

```diff
@@ -166,19 +168,25 @@ impl<T: Trait> Module<T> {
 	}
 
 	/// Set the new code.
-	fn set_code(new: Vec<u8>) -> Result {
+	pub fn set_code(new: Vec<u8>) -> Result {
 		storage::unhashed::put_raw(well_known_keys::CODE, &new);
 		Ok(())
 	}
 
 	/// Set some items of storage.
-	fn set_storage(items: Vec<KeyValue>) -> Result {
+	pub fn set_storage(items: Vec<KeyValue>) -> Result {
 		for i in &items {
 			storage::unhashed::put_raw(&i.0, &i.1);
 		}
 		Ok(())
 	}
 
+	/// Set the number of pages in the WebAssembly environment's heap.
+	pub fn set_heap_pages(pages: u64) -> Result {
+		storage::unhashed::put_raw(well_known_keys::HEAP_PAGES, &pages.encode());
```

Comment:
If we don't have any result in this function, we do we need to return one here?

Reply:
Just by looking around, how the other functions work in that file:

```rust
pub fn set_code(new: Vec<u8>) -> Result {
		storage::unhashed::put_raw(well_known_keys::CODE, &new);
		Ok(())
	}

	/// Set some items of storage.
	pub fn set_storage(items: Vec<KeyValue>) -> Result {
		for i in &items {
			storage::unhashed::put_raw(&i.0, &i.1);
		}
		Ok(())
	}

	/// Set the number of pages in the WebAssembly environment's heap.
	pub fn set_heap_pages(pages: u64) -> Result {
		storage::unhashed::put_raw(well_known_keys::HEAP_PAGES, &pages.encode());
		Ok(())
	}
```

I can assume this is the current convention. Not sure, if there a better reason. @gavofyork ?